externals/kyuubi-download/pom.xml

```diff
@@ -45,6 +45,7 @@
                         </goals>
                         <configuration>
                             <skip>${spark.archive.download.skip}</skip>
+                            <skipCache>true</skipCache>
```

Comment:
this seems irrelevant

Reply:
A small problem found during testing, need to submit a new issue to fix?